pkg/utils/selector.go

```diff
@@ -207,8 +217,8 @@ func CheckPodMeetSelector(pod v1.Pod, selector v1alpha1.SelectorSpec) (bool, err
 	return false, nil
 }
 
-// GeneratePods generate pods according to mode from pod list
-func GeneratePods(pods []v1.Pod, mode v1alpha1.PodMode, value string) ([]v1.Pod, error) {
+// FilterPodsByMode filter pods according to mode from pod list
```

Comment:
I think `FilterPodsByMode filters pods by mode` is better. What do you think?

Reply:
Thanks. I agree with you. I will fix it.